src/Command/Remote/DrushCommand.php

```diff
@@ -14,28 +14,25 @@
  * A command to proxy Drush commands on an environment using SSH.
  */
 #[RequireAuth]
-#[AsCommand(name: 'remote:drush', description: 'Run a Drush command remotely on a application\'s environment', aliases: ['drush', 'dr'])]
+#[AsCommand(name: 'remote:drush', description: 'Run a Drush command remotely on a Cloud Platform environment', aliases: ['drush', 'dr'])]
 final class DrushCommand extends SshBaseCommand {
 
   protected function configure(): void {
     $this
       ->setHelp('<fg=black;bg=cyan>Pay close attention to the argument syntax! Note the usage of <options=bold;bg=cyan>--</> to separate the drush command arguments and options.</>')
-      ->addArgument('alias', InputArgument::REQUIRED, 'Alias for application & environment in the format `app-name.env`')
+      ->acceptEnvironmentId()
       ->addArgument('drush_command', InputArgument::IS_ARRAY, 'Drush command')
       ->addUsage('<app>.<env> -- <command>')
       ->addUsage('myapp.dev -- uli 1')
       ->addUsage('myapp.dev -- status --fields=db-status');
   }
 
   protected function execute(InputInterface $input, OutputInterface $output): ?int {
-    $alias = $input->getArgument('alias');
-    $alias = $this->normalizeAlias($alias);
-    $alias = self::validateEnvironmentAlias($alias);
-    $environment = $this->getEnvironmentFromAliasArg($alias);
-
+    $environment = $this->determineEnvironment($input, $output);
```

Comment:
This is the meat of the fix, and has the side effect of changing the argument name from `alias` to `environmentId`. I'm 99% sure nobody will care or even notice.